docs/content/docs/user-docs/cross-account-resource-management.md

```diff
@@ -83,6 +83,212 @@ spec:
 EOF
 ```
 
+## OpenShift multiple AWS account pre-installation
+### Summary
+When ACK service controllers are installed via OperatorHub, a cluster administrator will need to perform the following pre-installation steps to provide the controller any credentials and authentication context it needs to interact with the AWS API.
+
+Rather than setting up a `ServiceAccount` like in the EKS instructions above, you need to use IAM users and policies. You will then set the required authentication credentials inside a `ConfigMap` and a `Secret`.
+
+The following directions will use the Elasticache controller as an example, but the instructions should apply to any ACK controller. Just make sure to appropriately name any values that include `elasticache` in them.
+
+Some notes for the following instructions:
+* The first user account, which is provided to the ACK controller and will assume the roles of any other accounts, will be identified with profile 000000000000
+* The second account will be identified with profile 111111111111
+* The instructions will only outline setting up a second account, but you should be able to follow the same instructions to set up user accounts and namespaces
+* Substitute actual ID values and namespaces to fit your actual environment values
+
+### Step 1: Create the user for the ACK controller and enable programmatic access
+
+Create a user with the `aws` CLI (named `ack-elasticache-service-controller` in our example):
+```bash
+aws --profile 000000000000 iam create-user --user-name ack-elasticache-service-controller
+```
+
+Make a note of the ARN for this user for use in the next step. An easy way to do so is to export it to an environment variable:
+```bash
+USER_ARN='arn:aws:iam::000000000000:user/ack-elasticache-service-controller'
+```
+
+Enable programmatic access for the user you just created:
+```bash
+aws --profile 000000000000 iam create-access-key --user-name ack-elasticache-service-controller
+```
+
+You should see output with important credentials:
+```json
+{
+    "AccessKey": {
+        "UserName": "ack-elasticache-service-controller",
+        "AccessKeyId": "00000000000000000000",
+        "Status": "Active",
+        "SecretAccessKey": "abcdefghIJKLMNOPQRSTUVWXYZabcefghijklMNO",
+        "CreateDate": "2021-09-30T19:54:38+00:00"
+    }
+}
+```
+
+This is the user that will end up representing our ACK service controller, which means these are the credentials we’ll eventually pass to our controller. Save or note `AccessKeyId` and `SecretAccessKey` in addition to the ARN you already saved for use in later steps.
```

Comment:
IRSA would not work for operators that are installed/managed via OLM. IRSA updates/changes the `env` of a deployment, which OLM then re-reonciles to prevent drift, thus removing the `env` added. This leaves the original deployment, and original pod(s) in place. This is why we went with adding in the below to the CSV

```
envFrom:
            - configMapRef:
                name: ack-user-config
                optional: true
            - secretRef:
                name: ack-user-secrets
                optional: true
```